contrib/libsass.spec

```diff
@@ -1,7 +1,7 @@
 Name:           libsass
-Version:        3.1.0
+Version:        %{version}
```

Comment:
`%{version}` is defined by what's in `Version:`, so I'm not sure how this works out. Maybe you can define it when building, like `rpmbuild --define 'version 1.2.3'`, but I'm not sure how it will work when the two things are clashing like this.